src/Cake.NuGet/Cake.NuGet.csproj

```diff
@@ -1,12 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Provides aliases (extension methods on Cake context) that support CI, build, unit tests, zip, signing, etc. for Cake.</Description>
```

Comment:
Any suggestion for this? @cake-build/cake-team 

Old packages on MyGet seems to use same description https://www.myget.org/feed/cake/package/nuget/Cake.NuGet/0.12.0